plugins/become/dzdo.py

```diff
@@ -90,7 +90,6 @@ def build_become_command(self, cmd, shell):
             flags = '%s -p "%s"' % (flags.replace('-n', ''), self.prompt)
 
         user = self.get_option('become_user')
```

Comment:
More a matter of style, but I would prefer to use a variable called `become_user` and another `user`, like in doas.py. This way we don't have to change the purpose and type of the `user` in the middle of the code.

Reply:
I've adjusted the code accordingly.